dbt/models/iasworld/schema/iasworld.aprval.yml

```diff
@@ -55,11 +55,14 @@ sources:
           - name: aprtot
             description: '{{ doc("column_aprtot") }}'
             tests:
-              - dbt_utils.accepted_range:
+              - accepted_range:
```

Comment:
This test started failing recently, so I switched it to use our new custom macro that allows selecting extra columns.